lima.nix

```diff
@@ -19,10 +19,13 @@
     };
 
     # system mounts
-    boot.loader.grub = {
-        device = "nodev";
-        efiSupport = true;
-        efiInstallAsRemovable = true;
+    boot = {
+        kernelParams = [ "console=tty0" ];
```

Comment:
What effect does this have when using QEMU?

Reply:
I verified that this change works with both QEMU and VZ.

Reply:
I cherry-picked this commit to `master`: 045cb1cdf454cd348e78e044772b4265c2959a4b

Reply:
Thank you.

Reply:
Thank you!